CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 2.26.0-rc.0 / 2021-03-24
+
+Notable changes:
+
+* Experimental query editor UI helps you to write correct PromQL queries, with auto-completion, syntax highlighting and linting. 
+* Native support for AWS authentication for `remote_write` to Amazon Managed Service for Prometheus. This eliminates the need for a sidecar.
+* Two experimental features have been added, these are [disabled by default and may change in the future](https://prometheus.io/docs/prometheus/latest/disabled_features/):
+  * If you enable `--enable-feature=exemplar-storage`, Prometheus will hold certain amount of scraped OpenMetrics' Exemplars in circular memory. It will also expose `/api/v1/query_exemplars` querying HTTP API.
+  * PromQL support for negative offset in queries via feature flag `--enable-feature=promql-negative-offset`. (eg. `sum_over_time(metric[1h:1m] offset -5m)`)
+* Prometheus is now built and supporting Go 1.16 (#8544). This reverts the memory release pattern added in Go 1.12. This makes common RSS usage metrics showing more accurate number for actual memory used by Prometheus. You can read more details [here](https://www.bwplotka.dev/2019/golang-memory-monitoring/).
+* This release changed the main development branch to `main`.
+
+* [CHANGE] Alerting: Using Alertmanager v2 API by default. #8626
+* [FEATURE] Remote: Add support for AWS SigV4 auth method for remote_write. #8509
+* [FEATURE] Scaleway Discovery: Add Scaleway Service Discovery. #8555
+* [FEATURE] PromQL: Allow negative offsets. Behind `--enable-feature=promql-negative-offset` flag. #8555
+* [FEATURE] **experimental** Exemplars: Add in-memory storage for exemplars. Behind `--enable-feature=exemplar-storage` flag. #6635
+* [FEATURE] UI: Add advanced auto-completion, syntax highlighting and linting to graph page query input. #8634
+* [ENHANCEMENT] Digital Ocean Discovery: Add `__meta_digitalocean_image` label. #8497
+* [ENHANCEMENT] Remote: Allow configuring retries on rate-limiting for remote_write. See [configuration docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) for details. #8477
+* [ENHANCEMENT] Authorization: Add support for specifying type of credentials with `Bearer` by default. #8512
+* [ENHANCEMENT] Prometheus/Promtool: Binaries are now printing help and usage to stdout instead of stderr. #8542
```

Comment:
```suggestion
* [CHANGE] Prometheus/Promtool: Binaries are now printing help and usage to stdout instead of stderr. #8542
```

As agreed in dev summit.

Reply:
I meant that it was agreed on dev summit to mark as a change, it was not needed to write it as is here ;)